src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandler.java

```diff
@@ -0,0 +1,245 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.ericsson.gerrit.plugins.eiffel.handlers;
+
+import java.io.File;
+import java.net.ConnectException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is the database handler class. At initiation it checks if the name given
+ * exist as a db file, if not it gets created together with required tables. At
+ * initiation a file name is given making it possible to store a file per
+ * project depending how the classes that uses this handler implements this
+ * function.
+ *
+ */
+public class DataBaseHandler {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DataBaseHandler.class);
+    private static final String EVENT_ID_KEY = "eventId";
+    private final String databaseFile;
+
+    /**
+     * Constructor that takes plugin directory path as String and plugin name as
```

Comment:
The javadoc seems a bit confusing as it is saying Plugin name where as the parameter says filename.
filename, N can be capitalized.

either use fileName or pluginName.